torch_glow/src/PyTorchModelLoader.cpp

```diff
@@ -734,6 +736,51 @@ llvm::Error PyTorchModelLoader::loadMin(const torch::jit::Node *ptNode) {
   return addGlowNodeValue(outputs[0], output);
 }
 
+llvm::Error PyTorchModelLoader::loadSoftMax(const torch::jit::Node *ptNode) {
+  auto inputs = ptNode->inputs();
+  auto outputs = ptNode->outputs();
+  RETURN_IF_ERR(checkInputAndOutputSizes(inputs, 3, outputs, 1));
+
+  // Indexes of aten::softmax inputs.
+  struct Inputs {
+    enum {
+      input = 0,
+      dim = 1,
+      dtype = 2,
+    };
+  };
+
+  glow::NodeValue in;
+  ASSIGN_VALUE_OR_RETURN_ERR(in, getGlowNodeValue(inputs[Inputs::input]));
+
+  // Dim.
+  glow::unsigned_t dim = 1;
+  if (hasGlowNodeValue(inputs[Inputs::dim])) {
+    auto dimHandle = glow::Handle<int32_t>::createInvalidHandle();
+    ASSIGN_VALUE_OR_RETURN_ERR(
+        dimHandle, getGlowConstantHandle<int32_t>(inputs[Inputs::dim]));
+    RETURN_IF_ERR(contractParamIfNeeded(dimHandle, dim));
+  }
+
+  // Dtype (optional).
+  glow::unsigned_t dtype = 1;
```

Comment:
This isn't used anywhere and also should use something other than 1 like an enum from somewhere if available not sure where torch.dtype can be found (maybe ScalarType?)